test/mailers/previews/publisher_mailer_preview.rb

```diff
@@ -18,14 +18,6 @@ def verification_done_internal
     PublisherMailer.verification_done_internal(Channel.first)
   end
 
-  def welcome
-    PublisherMailer.welcome(Publisher.first)
-  end
-
-  def welcome_internal
-    PublisherMailer.welcome_internal(Publisher.first)
-  end
-
```

Comment:
(Outside the scope of this PR) I noticed that a lot of methods are missing from the `PublisherMailerPreview` and some like `verification_done` raise an error.  I will probably open a separate issue to address this.